.github/workflows/build.yml

```diff
@@ -60,6 +84,26 @@ jobs:
             echo "Reusing repo mirror @ $REPO_MIRROR" ;
           fi ;
 
+      - name: Cache save gitee repositories
```

Comment:
We save these in all cases in the current workflow. Would it make sense to only save them again when changed from the earlier restore?

Reply:
it should be saving only when the manifest has changed because I set the key to `cache-repos-${{ hashFiles('**/*.xml') }} ` which does a hash of the manifests